docs/application-setup.md

```diff
@@ -22,7 +22,8 @@ Choose the framework to build your application:
 [Application setup without a framework](#application-setup-for-an-application-not-using-a-framework)  
 [Angular 6](#application-setup-for-angular-6)  
 [SAPUI5/OpenUI5](#application-setup-for-sapui5openui5)  
-[VUE.JS](#application-setup-for-vuejs)
+[VUE.JS](#application-setup-for-vuejs)  
+[React.JS](#application-setup-for-reactjs)  
```

Comment:
```suggestion
[React](#application-setup-for-react)  
```
I think the "official" name for the framework is only React (while Vue also includes the ".js" in their name, at least according to their website).